fmcadc4/fmcadc4.c

```diff
@@ -238,9 +238,9 @@ int main(void)
 
 	xcvr_getconfig(&ad9680_xcvr);
 	ad9680_xcvr.reconfig_bypass = 0;
-	ad9680_xcvr.ref_clock_khz = 500000;
-	ad9680_xcvr.lane_rate_kbps = ad9680_0_param.lane_rate_kbps;
-	ad9680_xcvr.dev.qpll_enable = 1;
+	ad9680_xcvr.ref_rate_khz = 500000;
+	ad9680_xcvr.lane_rate_khz = ad9680_0_param.lane_rate_kbps;
```

Comment:
this seems a bit weird now  ; i.e: assiging rate_kbps to rate_khz ; 
maybe also update that field as well ?